code/qc_utils.py

```diff
@@ -73,13 +74,30 @@ def load_processing_metadata(processing_json):
     return Processing(**processing_dict)
 
 
+def _get_surface_channel(recording: si.BaseRecording, recording_name: str, channel_labels: np.ndarray) -> int | None:
```

Comment:
@arjunsridhar12345 the `recording_name` is only used for logging. I would just remove it so you can also remove it from `plot_raw_data` and `plot_psd`